### ISSUE TITLE
paper equations vs code

### ISSUE THREAD

**Issue by anantiitml** [chunyuanY/RumorDetection#7]
Hi,

I have read your paper it says it derives the conv features first from the w2v embedding and then applied multi-head attention over that to arrive at final message  embeddings and then applied GRE to  obtain another representation. but in your code directly conv features are fed for classification 
Please clarify:

        conv_feature = torch.cat(conv_block, dim=1)
        features = self.dropout(conv_feature)

        a1 = self.relu(self.fc1(features))
        d1 = self.dropout(a1)

        output = self.fc2(d1)
        return output

where fc2  -----> self.fc2 = nn.Linear(300, config['num_classes'])

I am writing the survey paper to include your results please verify the code or give me the updated code as per paper.

**Comment by vllgle** [comment]
> Hi,
> 
> I have read your paper it says it derives the conv features first from the w2v embedding and then applied multi-head attention over that to arrive at final message embeddings and then applied GRE to obtain another representation. but in your code directly conv features are fed for classification
> Please clarify:
> 
> ```
>     conv_feature = torch.cat(conv_block, dim=1)
>     features = self.dropout(conv_feature)
> 
>     a1 = self.relu(self.fc1(features))
>     d1 = self.dropout(a1)
> 
>     output = self.fc2(d1)
>     return output
> ```
> 
> where fc2 -----> self.fc2 = nn.Linear(300, config['num_classes'])
> 
> I am writing the survey paper to include your results please verify the code or give me the updated code as per paper.

**Comment by ethan-phu** [comment]
Hello, I want to know how the r value in target:r in *graph.txt in the data set is obtained. Is it the value normalized by time?
